xinference/model/llm/llm_family.json

```diff
@@ -1322,6 +1322,66 @@
         ],
         "model_id": "TheBloke/CodeLlama-34B-fp16",
         "model_revision": "f91d0cf7fc338cdc726f9c72d5ea15fe51bb16e9"
+      },
+      {
+        "model_format": "ggufv2",
+        "model_size_in_billions": 7,
+        "quantizations": [
+          "Q2_K",
```

Comment:
A problem here. 

The previous naming of quantizations are like `q_XX`, but recent naming became `Q_XX`. This inconcsistency could confuse users since they can hardly konw the correct quantization name to use.

Maybe we should have our own names for ggml quantizations. What do you think?

Reply:
I prefer to use what llamacpp use, If there is a new quantization that does not match previous one, compatibility will become a major issue.